packages/documentation-site/patternfly-docs/content/get-started/upgrade.md

```diff
@@ -51,15 +58,17 @@ Where we can, we offer a suite of codemods to simplify and streamline your upgra
 
 7. Run codemods multiple times to ensure that all issues are flagged and addressed.
 
-### 2. Review and update variable and class names
+### 3. Review and update variable and class names
 
 PatternFly 6 supports our new design token system, which changes variable names across PatternFly. These changes mean that **all** existing CSS overrides must be updated (or removed), because they will be targeting outdated styles and will no longer work. 
```

Comment:
Should we add something reminding them that in V5, a version prefix was added to all CSS variables and classes, so the first step is to update this prefix from "v5" to "v6" wherever they've use it. (is there a codemod for this?)
Then move into the new design token system information.